### ISSUE TITLE
Error with cached routes

### ISSUE THREAD

**Issue by Gamerboy59** [cp6/my-idlers#102]
After I cached routes with `php artisan route:cache` I'm getting a 500 error for `HEAD not supported for /.`. I suppose something is wrong or outdated at the routes config in the `routes/*` folder but I couldn't find it at a glance. Only the core / root path appears to be effected.

**Comment by cp6** [comment]
Shared hosting environment?

**Comment by Gamerboy59** [comment]
Yes, Plesk.

**Comment by cp6** [comment]
Ok I cant really find any answers for this currently, was it working previously in this environment?

**Comment by Gamerboy59** [comment]
No, this error appears for a while already. I couldn't find an answer but it's likely this is related to #57. I tested another Laravel app (not my-idlers) and it's working fine when caching routes hence I guess it's something with this app and not my webserver. Additionally, I was just testing it on my other cPanel host but same error. It works as long as the routes aren't cached so I removed this command from my git provisioneer plugin.

**Comment by cp6** [comment]
Are you able to please show the output of `php artisan route:list` ?
